draft-kuehlewind-iab-rfc4053bis.md

```diff
@@ -434,18 +434,22 @@ work item requires a charter change.
 
 There is, of course, no requirement that IETF perform the action that
 was requested.  But the request should always be taken seriously, and
-generally a response is anticipated.  The originating organization should always be
-informed of what, if anything, the IETF has decided to do in response
-to the request.  If the IETF decides not to honor the request, or to
-honor it with modifications, the response should include the reasons
-and, if applicable, the alternate course of action.
+generally a response is anticipated with the specific deadline.  The
+originating organization should always be informed of what, if anything,
+the IETF has decided to do in response to the request.  If the IETF
+decides not to honor the request, or to honor it with modifications,
+the response should include the reasons and, if applicable, the alternate
+course of action.
 
 For tasks that require a great deal of time, it may be necessary that
 several liaison statements be sent back to the originating
 organization to report the status of the work and the anticipated
 completion time.  The first of these liaison statements should be
 generated by the deadline indicated in the incoming liaison
-statement.
+statement. For tasks with requested deadline, one or multiple notifications
+or reminders to call for response can be triggered. If the deadline for
+specific task is not respected, responding to incoming liaison statement
+should be escalated into the sponsored AD or liaison managers.
```

Comment:
Not sure we can put this like this because that's again a tooling requirement that we decided to remove from this document.

@sureshkrishnan what did you have on your mind for this?